shared/src/persistence/s3/getDownloadPolicyUrl.ts

```diff
@@ -5,12 +5,17 @@
  * @param {string} providers.key the key of the document to get
  * @returns {Promise<any>} the promise of the call to the storage client
  */
-exports.getDownloadPolicyUrl = ({
+export const getDownloadPolicyUrl = ({
   applicationContext,
   filename,
   key,
-  useTempBucket,
-}) => {
+  useTempBucket = false,
+}: {
+  applicationContext: IApplicationContext;
+  filename?: string;
+  key: string;
+  useTempBucket?: boolean;
```

Comment:
For my understanding, since `useTempBucket` is defaulted to false, does it also need to be marked as optional?

Reply:
yeah, if you don't set it as optional, typescript will complain expecting you to set where ever you invoke this function